RZBluetooth/RZBPeripheral.m

```diff
@@ -155,8 +162,10 @@ - (void)clearNotifyBlockForCharacteristicUUID:(CBUUID *)characteristicUUID
 
     // Remove the completion block immediately to behave consistently.
     // If anything here is nil, there is no completion block, which is fine.
+    // (Nope! If characteristic is nil, -setNotifyBlock:forCharacteristicUUID: would crash! CSP 9/9/2016)
     CBService *service = [self.centralManager serviceForUUID:serviceUUID onPeripheral:self.corePeripheral];
     CBCharacteristic *characteristic = [service rzb_characteristicForUUID:characteristicUUID];
+    // ...but nil characteristic was NOT handled here. CSP 9/9/2016
     [self setNotifyBlock:nil forCharacteristicUUID:characteristic.UUID];
```

Comment:
I'd prefer to fix this crash right here. We should be able to change `characteristic.UUID` to `characteristicUUID`, and this should ensure that nil doesn't occur.